### PR TITLE
Add basic UI Kit themed error pages

### PR DESCRIPTION
**Pull request by maxious**
screen shot 2016-08-09 at 11 25 24 am

### CODE REVIEW

**Comment by tessereth** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

**Comment by maxious** [comment]
rebase'd

File: app/views/errors/change_rejected.html.haml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+= content_for :title do
+  The change you wanted was rejected (422)
+
+%article.content-listing.callout--calendar-event
```
Comment (klepas) [inline app/views/errors/change_rejected.html.haml]: Odd markup callout choice? :S

**Comment by klepas** [comment]
Why go with the calendar style callout?

**Pull request merged**